べっちさん課題/index.html

```diff
@@ -1,33 +1,22 @@
-<!DOCTYPE html>
-<html lang="ja">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>べっちさんの課題</title>
-</head>
-<body>
-  <script>
+<script>
   'use strict';
 
   function fizzBuzz(){
   for(let i =1; i < 100 ; i++){
 
     if(i % 3 === 0 && i % 5 === 0){
-      console.log("FizzBuzz" + i);
-    }else if(i % 3 === 0){
-      console.log("Fizz" + i);
+        console.log("FizzBuzz" + i);
+        }else if(i % 3 === 0){
```

Comment:
フォーマットをかけると以下のようになると思ったんですが、なりませんでしたか？
```suggestion
        } else if (i % 3 === 0){
```

Reply:
Visual Studio Code の機能を使ってフォーマット、検索して
新たにPrettier - Code formatterという拡張機能にたどり着きました！
ご指摘のおかげで拡張機能に気がつくことができました
ありがとうございます！！